platform/mds/mds-test-bundle/src/main/java/org/motechproject/mds/test/service/TestRevertFromTrashService.java

```diff
@@ -0,0 +1,8 @@
+package org.motechproject.mds.test.service;
+
+import org.motechproject.mds.service.MotechDataService;
+import org.motechproject.mds.test.domain.RevertFromTrash;
+
+
+public interface TestRevertFromTrashService extends MotechDataService<RevertFromTrash> {
```

Comment:
this service also should be in the new package and add osgi reference for this service